src/backend/drakula/database.py

```diff
@@ -1,74 +1,96 @@
-            port=port,
-            user=user,
-            password=password,
-            database="flight_game",
-            autocommit=True,
-        )
+        try:
+            self.connection = connect(
+                host=host,
+                port=port,
+                user=user,
+                password=password,
+                database="flight_game",
+                autocommit=True,
+                auth_plugin="mysql_native_password",
+            )
+        except Error as e:
+            print(f"Error: {e}")
+            self.connection = None
 
-    def get_airports(self, seed: Optional[int] = None, amount: int = DEFAULT_AIRPORT_AMOUT) -> list[Airport]:
+    def get_airports(
+        self, seed: Optional[int] = None, amount: int = DEFAULT_AIRPORT_AMOUT
+    ) -> list[Airport]:
         cursor = self.connection.cursor(dictionary=True)
         cursor.execute(
-            f"""SELECT
+            """SELECT
                 name, latitude_deg, longitude_deg, iso_country
-                FROM airport ORDER BY RAND({seed or ''}) LIMIT {amount or DEFAULT_AIRPORT_AMOUT}"""
+                FROM airport ORDER BY RAND(%s) LIMIT %s""",
+            (seed, amount)
         )
         return list(map(lambda args: Airport(**args), cursor.fetchall()))
-    
+
     def set_save(self, id, json):
         if not self.connection:
             print("Database connection error!")
             return None
         cursor = self.connection.cursor()
-        
-        cursor.execute(
-            f"""INSERT INTO games (id) VALUE (%s, ?, %s)""",
-            (id, json)
-        )
-        print(f"Insert new player complete!")
 
-    
-    def get_save(self,game_id: str, amount: int = DEFAULT_AIRPORT_AMOUT) -> list[dict]:
+        try:
+            cursor.execute(
+                """INSERT INTO games (id, json_data) VALUES (%s, %s)""", (id, json)
+            )
+            print(f"Insert new player complete!")
+        except Error as e:
+            print(f"Error: {e}")
+
+    def get_save(self, game_id: str, amount: int = DEFAULT_AIRPORT_AMOUT) -> list[dict]:
         if not self.connection:
             print("Database connection error!")
             return None
         cursor = self.connection.cursor(dictionary=True)
-        cursor.execute(
-            "SELECT * FROM games WHERE id = %s",
-            (game_id,)
+        cursor.execute("SELECT * FROM games WHERE id = %s", (game_id,))
+
+        return cursor.fetchall()
+
+
+def make_db() -> Optional[Database]:
+    try:
+        db = Database(
+            host=getenv("DRAKULA_V2_MARIADB_HOST") or "127.0.0.1",
+            port=int(getenv("DRAKULA_V2_MARIADB_PORT") or "3306"),
+            password=getenv("DRAKULA_V2_MARIADB_PASSWORD"),
+            user=getenv("DRAKULA_V2_MARIADB_USER"),
         )
 
-        return cursor.fetchall()        
+        if not db.connection:
+            print("Database connection error!")
+            return None
+        full_path = "query.sql"
```

Comment:
Again, no reason to load it as a file. We can just inline the query.

Reply:
done